### PR TITLE
:recycle: replace turbo by mocha

### PR DESCRIPTION
**Pull request by corinnekrych**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -179,12 +157,10 @@ module.exports = function(grunt) {
   // Testing tasks
   grunt.registerTask('test', ['shell:unit', 'shell:accept']);
   grunt.registerTask('unit', ['shell:unit']);
-  grunt.registerTask('accept', ['env:accept', 'shell:accept']);
 
   // Coverate tasks
-  grunt.registerTask('coverage', ['shell:coverage_unit', 'shell:coverage_accept']);
+  grunt.registerTask('coverage', ['shell:coverage_unit']);
   grunt.registerTask('coverage-unit', ['shell:coverage_unit']);
```
Comment (jcesarmobile) [inline Gruntfile.js]: coverage and coverage-unit do the same now, so you can probably delete coverage-unit

**Comment by corinnekrych** [comment]
@jcesarmobile mind a second 👁  ?

File: Gruntfile.js [hunk 2]
```diff
@@ -122,20 +113,7 @@ module.exports = function(grunt) {
...
-        options: {
-          stdout: true,
-          stderr: true,
-          failOnError: true
-        },
-        command: [
-          'rm -rf coverage cov-accept',
-          'env NODE_PATH=. ./node_modules/.bin/istanbul cover --dir cov-accept ./node_modules/.bin/turbo -- --setUp=test/accept/server.js --tearDown=test/accept/server.js test/accept',
+          'env NODE_PATH=. ./node_modules/.bin/istanbul cover --dir cov-unit ./node_modules/.bin/_mocha -- -u exports --recursive test/unit/test*.js',
           './node_modules/.bin/istanbul report',
```
Comment (jcesarmobile) [inline Gruntfile.js]: remove this as the coverage report is already generated on `cov-unit`

File: package.json [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
+  "name": "feedhenryhealthmonitoring",
+  "version": "0.1.2",
   "scripts": {
-    "test": "grunt jshint",
-    "start": "node application.js"
+    "start": "node application.js",
+    "serve": "grunt serve",
+    "unit": "grunt test",
+    "test": "npm run unit",
+    "coverage-unit": "grunt coverage"
```
Comment (jcesarmobile) [inline package.json]: call it `coverage` as on other templates

File: README.md [hunk 4]
```diff
@@ -65,19 +65,33 @@ For local development, the default value for the MongoDB connection string is de
...
+```
+### test coverage
+As pre-requisites:
+```
+npm run serve
+mongod
+```
+Run unit test:
+``` 
+npm run coverage-unit
```
Comment (jcesarmobile) [inline README.md]: run with `npm run coverage` as other templates, needs to change the package.json, see the other comment

File: package.json [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
-  "version": "0.1.1",
+  "name": "feedhenryhealthmonitoring",
+  "version": "0.1.2",
   "scripts": {
-    "test": "grunt jshint",
-    "start": "node application.js"
+    "start": "node application.js",
+    "serve": "grunt serve",
+    "unit": "grunt test",
+    "test": "npm run unit",
```
Comment (jcesarmobile) [inline package.json]: why make `test` command to run `npm run init` and then `unit` run `grunt test` and not make `test` call `grunt test` directly?

**Comment by trepel** [comment]
@jcesarmobile @corinnekrych reviewing

File: test/accept/test_checkList.js [hunk 6]
```diff
@@ -1,13 +0,0 @@
...
-      .maximizeWindow()
-      .url(client.launch_url)
-      .waitForElementVisible("body", 1000)
-      .assert.title("FeedHenry HMS")
-      .waitForElementNotVisible("#nprogress",2000)
-      .waitForElementVisible("#monitorlist",1000)
-      .assert.elementPresent("#monitorlist_head")
-      .end()
-  }
-};
```
Comment (trepel) [inline test/accept/test_checkList.js]: Why was this acceptance test removed ?

Comment (corinnekrych) [inline test/accept/test_checkList.js]: @trepel this test was deleted as I think t was never run

**Comment by corinnekrych** [comment]
@jcesarmobile fixed as suggested and rebased/squashed

**Pull request merged**